src/cart/test/util/cart_logtest.py

```diff
@@ -117,10 +117,13 @@ class LogError(LogCheckError):
                      'mgmt_svc_name_cb': ('s'),
                      'pool_prop_default_copy': ('entry_def->dpe_str'),
                      'pool_iv_prop_g2l': ('prop_entry->dpe_str'),
-                     'pool_iv_value_alloc_internal': ('sgl->sg_iovs[0].iov_buf'),
+                     'pool_iv_value_alloc_internal':
+                             ('sgl->sg_iovs[0].iov_buf'),
```

Comment:
(pylint-bad-continuation) Wrong hanging indentation in dict value.

Reply:
These entries are never going to fit within the 80 character limit and continuations like this just make the code block longer so I've just been doing them all one-per-line.

Reply:
Will revert